### PR TITLE
#568 : cleanup unnecessary logs

### PR DESCRIPTION
**Pull request by adityankannan-tw**
To improve the speed of transfer, 

Removed the debug logs that print large amount of VC data.
Removed logs added to print SHA for each data chunk in both iOS and android side.
Removed logs to print the complete encrypted VC data for both iOS and android side.
Removed logs printing any encryption key in both iOS and Android side.

Changed important print statements to info os_logs and also commented out print statements that are used for debugging.

### CODE REVIEW

File: ios/Wallet/Wallet.swift [hunk 1]
```diff
@@ -68,7 +68,7 @@ class Wallet: NSObject {
 
         if (encryptedData != nil) {
             //print("Complete Encrypted Data: \(encryptedData!.toHex())")
-            print("Sha256 of Encrypted Data: \(encryptedData!.sha256())")
+            os_log(.info, "Sha256 of Encrypted Data: %{public}@ ", (encryptedData!.sha256()))
```
Comment (ravikp) [inline ios/Wallet/Wallet.swift]: Encrypted data shouldn't be printed.

File: ios/Openid4vpBle/Openid4vpBle.swift [hunk 2]
```diff
@@ -64,19 +64,19 @@ class Openid4vpBle: RCTEventEmitter {
...
+            //print("EXCHANGE-SENDER-INFO")
             callback([])
             Wallet.shared.writeToIdentifyRequest()
         case "send-vc":
             callback([])
-            print(">> raw message size", messageComponents[1].count)
+            os_log(.info, ">> raw message size : %{public}d", messageComponents[1].count)
             Wallet.shared.sendData(data: messageComponents[1])
         default:
             print("DEFAULT SEND: MESSAGE:: ", message)
```
Comment (ravikp) [inline ios/Openid4vpBle/Openid4vpBle.swift]: These state transitions are good to be printed.

File: ios/Wallet/Wallet.swift [hunk 3]
```diff
@@ -109,19 +109,19 @@ class Wallet: NSObject {
...
         if let data {
             let connStatusID = Int(data[0])
             if connStatusID == 1 {
-                print("con statusid:", connStatusID)
+                //print("con statusid:", connStatusID)
                 destroyConnection()
             }
         } else {
-            print("weird reason!!")
+            //print("weird reason!!")
```
Comment (ravikp) [inline ios/Wallet/Wallet.swift]: This error condition needed to be printed with proper reason using os_log.

Comment (ravikp) [reply ios/Wallet/Wallet.swift]: Otherwise the error is escaped.

File: ios/ble/Utility/TransferHandler.swift [hunk 4]
```diff
@@ -79,13 +79,13 @@ class TransferHandler {
...
-        print("transmission report requested")
+        os_log(.info, "transmission report requested")
     }
 
     private func handleTransmissionReport(report: Data) {
         let r = TransferReport(bytes: report)
-        print(" got the transfer report type \(r.type)")
-        print("missing pages: ", r.totalPages)
+        //print(" got the transfer report type \(r.type)")
+        //print("missing pages: ", r.totalPages)
```
Comment (ravikp) [inline ios/ble/Utility/TransferHandler.swift]: These small logs can be kept.

**Review by ravikp — CHANGES_REQUESTED** [review]
Given a few comments on the log lines. Request you to please resubmit them.

File: android/src/main/java/io/mosip/tuvali/transfer/Chunker.kt [hunk 5]
```diff
@@ -34,8 +34,7 @@ class Chunker(private val data: ByteArray, private val maxDataBytes: Int = DEFAU
   private fun chunk(seqNumber: Int): ByteArray {
     val fromIndex = seqNumber * effectivePayloadSize
 
-    return if (seqNumber == (totalChunkCount - 1).toInt() && lastChunkByteCount > 0) {
-      Log.d(logTag, "fetching last chunk")
+    return if (seqNumber == (totalChunkCount - 1).toInt() && lastChunkByteCount > 0) { Log.d(logTag, "fetching last chunk")
```
Comment (vharsh) [inline android/src/main/java/io/mosip/tuvali/transfer/Chunker.kt]: _please see if this code-change is correctly formatted as per current convention_

File: ios/Wallet/Wallet.swift (hunk 1 above)
Comment (adityankannan-tw) [reply ios/Wallet/Wallet.swift]: commented it

File: ios/Openid4vpBle/Openid4vpBle.swift (hunk 2 above)
Comment (adityankannan-tw) [reply ios/Openid4vpBle/Openid4vpBle.swift]: uncommented.

File: ios/Wallet/Wallet.swift (hunk 3 above)
Comment (adityankannan-tw) [reply ios/Wallet/Wallet.swift]: Yes. uncommented.

File: ios/ble/Utility/TransferHandler.swift (hunk 4 above)
Comment (adityankannan-tw) [reply ios/ble/Utility/TransferHandler.swift]: uncommented it.

File: android/src/main/java/io/mosip/tuvali/transfer/Chunker.kt (hunk 5 above)
Comment (adityankannan-tw) [reply android/src/main/java/io/mosip/tuvali/transfer/Chunker.kt]: formatted it.

File: android/src/main/java/io/mosip/tuvali/wallet/transfer/TransferHandler.kt [hunk 6]
```diff
@@ -123,7 +123,7 @@ class TransferHandler(looper: Looper, private val central: Central, val serviceU
     if (report.type == TransferReport.ReportType.SUCCESS) {
       currentState = States.TransferVerified
       transferListener.onResponseSent()
-      Log.d(logTag, "handleMessage: Successfully transferred vc in ${System.currentTimeMillis() - responseStartTimeInMillis}ms")
+      //Log.d(logTag, "handleMessage: Successfully transferred vc in ${System.currentTimeMillis() - responseStartTimeInMillis}ms")
```
Comment (Alka1703) [inline android/src/main/java/io/mosip/tuvali/wallet/transfer/TransferHandler.kt]: Make this as a info log. Do not comment this log. It gives us an idea on the transfer time.

**Review by Alka1703 — CHANGES_REQUESTED** [review]

File: android/src/main/java/io/mosip/tuvali/verifier/Verifier.kt [hunk 7]
```diff
@@ -246,7 +246,7 @@ class Verifier(
 
   override fun onResponseReceived(data: ByteArray) {
     Log.i(logTag, "Sha256 of complete encrypted data: ${Util.getSha256(data)}")
```
Comment (Alka1703) [inline android/src/main/java/io/mosip/tuvali/verifier/Verifier.kt]: Can you please comment this?

File: android/src/main/java/io/mosip/tuvali/wallet/Wallet.kt [hunk 8]
```diff
@@ -330,7 +330,7 @@ class Wallet(
     try {
       val encryptedData = secretsTranslator?.encryptToSend(compressedBytes)
       if (encryptedData != null) {
-        Log.d(logTag, "Complete Encrypted Data: ${Hex.toHexString(encryptedData)}")
+        //Log.d(logTag, "Complete Encrypted Data: ${Hex.toHexString(encryptedData)}")
         Log.i(logTag, "Sha256 of Encrypted Data: ${Util.getSha256(encryptedData)}")
```
Comment (Alka1703) [inline android/src/main/java/io/mosip/tuvali/wallet/Wallet.kt]: Can you comment this?

File: ios/ble/central/Central.swift [hunk 9]
```diff
@@ -24,20 +24,20 @@ class Central: NSObject, CBCentralManagerDelegate {
...
 
     deinit {
-        os_log("Central is DeInitializing")
+        os_log(.info, "Central is DeInitializing")
     }
 
     func scanForPeripherals() {
         centralManager.scanForPeripherals(withServices: [Peripheral.SERVICE_UUID], options: [CBCentralManagerScanOptionAllowDuplicatesKey: false])
-        os_log("scanning happening :::::::")
+        os_log(.info, "scanning happening :::::::")
```
Comment (Alka1703) [inline ios/ble/central/Central.swift]: Please remove colons

**Review by Alka1703 — CHANGES_REQUESTED** [review]

File: android/src/main/java/io/mosip/tuvali/verifier/Verifier.kt (hunk 7 above)
Comment (adityankannan-tw) [reply android/src/main/java/io/mosip/tuvali/verifier/Verifier.kt]: commented.

File: android/src/main/java/io/mosip/tuvali/wallet/Wallet.kt (hunk 8 above)
Comment (adityankannan-tw) [reply android/src/main/java/io/mosip/tuvali/wallet/Wallet.kt]: commented it.

File: android/src/main/java/io/mosip/tuvali/wallet/transfer/TransferHandler.kt (hunk 6 above)
Comment (adityankannan-tw) [reply android/src/main/java/io/mosip/tuvali/wallet/transfer/TransferHandler.kt]: done

File: ios/ble/central/Central.swift (hunk 9 above)
Comment (adityankannan-tw) [reply ios/ble/central/Central.swift]: done.

**Pull request merged**